wallix/README.adoc

```diff
@@ -0,0 +1,69 @@
+# Plugin for Wallix Bastion and Rudder
+
+This project is part of Rudder - IT infrastructure Automation and Compliance. See: http://rudder.io for more information.
+
+See repository main README for information about licensing, contribution, and other general information.
+
+// Everything after this line goes into Rudder documentation
+// ====doc====
+[Wallix Bastion-plugin]
+
+= Bastion Wallix integration plugin with Rudder
```

Comment:
Should just be "Wallix Bastion" as it will be used as title in the plugin list in the doc